docs/sphinx/api/rest/.openapi.yaml

```diff
@@ -0,0 +1,187 @@
+swagger: "2.0"
```

Comment:
Do these files have top banners?

Might be good to add a comment to summarize what this file is for either way though.

Reply:
I've added a top banner with comment about the purpose of this file.